app/src/app/api/v0/organizations/[organization]/invitations/route.ts

```diff
@@ -43,6 +45,28 @@ export const POST = apiHandler(async (req, { params, session }) => {
     throw new createHttpError.NotFound("organization-not-found");
   }
 
+  const existingOrgAdmins = await OrganizationAdmin.findAll({
+    include: [
+      {
+        model: User,
+        as: "user",
+        where: {
+          email: {
+            [Op.in]: validatedData.inviteeEmails,
+          },
+        },
+      },
+    ],
+  });
+
+  if (existingOrgAdmins.length > 0) {
+    throw new createHttpError.BadRequest(
+      `The following users are already admins for another organization: ${existingOrgAdmins
+        .map((admin) => admin.user.email)
+        .join(", ")}`,
```

Comment:
Here it would be better to do the map/ join outside of the string and save it to a variable instead imo